table/block_based/full_filter_block.cc

```diff
@@ -109,8 +109,11 @@ Slice FullFilterBlockBuilder::Finish(
   *status = Status::OK();
   if (any_added_) {
     any_added_ = false;
-    Slice filter_content =
-        filter_bits_builder_->Finish(filter_data ? filter_data : &filter_data_);
+    Slice filter_content = filter_bits_builder_->Finish(
+        filter_data ? filter_data : &filter_data_, status);
+    if (status->ok()) {
+      *status = filter_bits_builder_->PostVerify(filter_content);
```

Comment:
TODO:
Need to move the PostVerify after computing block checksum based on the task update

Reply:
Done